src/cryptoadvance/specter/templates/settings/tor_settings.jinja

```diff
@@ -0,0 +1,76 @@
+{% extends "base.jinja" %}
+{% block main %}
+	<form action="?" method="POST" onsubmit="showPacman()">
+		<h1 id="title" class="settings-title">Tor settings - Specter Desktop {{ current_version }}</h1>
+		{% from 'settings/components/settings_menu.jinja' import settings_menu %}
+		{{ settings_menu('tor', current_user) }}
+		<div class="card" style="margin: 20px auto;">
+			<h1>
+			<img style="width: 25px; margin-right: 7px; vertical-align: middle;" src="{{ url_for('static', filename='img/tor.svg') }}"/><span style="vertical-align: bottom; margin-right: 10px;">
+			Tor configurations
+			</h1>
+			Use Tor proxy running at: 
+			<input id="proxy-url" name="proxy_url" value="{{ proxy_url }}" type="url" placeholder="i.e. socks5://127.0.0.1:9050" />
+			<br>
+            <p class="note">
+			Default for Tor daemon: socks5://127.0.0.1:9050<br>
+			Default for Tor Browser: socks5://127.0.0.1:9150
+			</p>
+			Tor control port:
+			<p class="note">
+			Restart Specter for change of the control port to take effect.
+			</p>
+			<input name="tor_control_port" value="{{ tor_control_port }}" type="text" placeholder='Set "default" for default or specify port to use.' />
```

Comment:
The placeholder text is not visible if the field is pre-populated with "default". And default is also a very strange value. 
Maybe we can we keep it empty and do the placeholder like:
"Set to custom port here, if kept empty the default (9051) is used"